src/main/java/org/galatea/starter/ApiError.java

```diff
@@ -0,0 +1,21 @@
+package org.galatea.starter;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+
+import org.springframework.http.HttpStatus;
+
+@Getter
+@ToString
+@AllArgsConstructor
+public class ApiError {
```

Comment:
Should this move to the service package? The root package is mostly for the application starting point and configs.